docs/riff_streaming_stream.md

```diff
@@ -8,7 +8,9 @@ title: "riff streaming stream"
 
 ### Synopsis
 
-<todo>
+Streams are Custom Resources encapsulating the underlying message broker's streams.
```

Comment:
"Streams are streams" seems a little "too easy". Maybe should focus on the concept and what that represents, irrespective of how it's often implemented (could be something else than a message broker if we extrapolate).

Reply:
something something addressable message channels something?

Reply:
`Custom Resources` is not a proper noun. I'm not sure sure it's worth mentioning. Just about everything the cli interacts with is a custom resource.

Reply:
`Stream` is a riff concept. `stream` is not. Should I phrase it that way?

Reply:
The best I have so far is:
```
A stream encapsulates an addressable message channel (typically a message broker's topic). It can be mapped to a function input or output stream.
```
Let me know if that works.